src/utils/parse-arguments-into-options.ts

```diff
@@ -73,7 +83,19 @@ export async function parseArgumentsIntoOptions(rawArgs: Args): Promise<RawOptio
     project,
     install: hasInstallRelatedFlag ? install || !skipInstall : null,
     dev,
-    extensions: null, // TODO add extensions flags
     externalExtension: extension,
+    solidityFramework,
   };
 }
+
+const SOLIDITY_FRAMEWORK_OPTIONS = ["hardhat", "foundry", "none"];
+
+function solidityFrameworkHandler(value: string) {
+  const lowercasedValue = value.toLowerCase();
+  if (SOLIDITY_FRAMEWORK_OPTIONS.includes(lowercasedValue)) {
+    return lowercasedValue as SolidityFramework | "none";
+  }
```

Comment:
If the user has spelling mistake the user goes forward and gives the option: 

![Screenshot 2024-06-19 at 8 32 31 PM](https://github.com/scaffold-eth/create-eth/assets/80153681/ab40b27b-ef58-45ad-9e7b-d58871b90d50)


Alternate way will be, if `throw new Error("Wrong option provided, options can be hardhat | foundry | none");` the cli exits with : 

![Screenshot 2024-06-19 at 8 38 34 PM](https://github.com/scaffold-eth/create-eth/assets/80153681/46516e36-8b0c-47b8-99c3-bff82be35ca0)


lol both seems good !

Reply:
voting for first, because you don't need to restart the process because of the typo

Reply:
> you don't need to restart the process because of the typo

Yup agree!